functions.js

```diff
@@ -1 +1,126 @@
-// YOUR CODE HERE
+// maxOrMin(num1, num2, max)
+
+// Define a function maxOrMin that takes three parameters: two numbers and a boolean.
+// Have it return the larger of the two numbers if the boolean is true, otherwise have it return the
+// lesser of the numbers.
+
+function maxOrMin(num1, num2, max) {
+    if (max) {
+      return (num1 > num2)?num1:num2;
+    } else {
+      return (num1 < num2)?num1:num2;
+    }
+}
+
+// sillySum(arr)
+
+var sillySum = [2, 4, 5, 6];
+var result = 0;
+
+for (i = 0; i < sillySum.length; i++) {
+  result += i * sillySum[i];
+}
+
+console.log(result);
+
+// sillySum(arr) - Alt: return new array with each number times it's index
+
+var oldArray = [1, 2 ,3, 4, 5];
+
+function sillySum(oldArray) {
+    return oldArray.map(function callback(element, index) {
+      return element * index;
+    });
+  }
+
+sillySum(oldArray);
+
```

Comment:
Good work overall. Your 1st sillySum solution should be in function form.  The second returns an array instead of the sum of its elements